code/datums/components/healing_reduction.dm

```diff
@@ -33,15 +33,18 @@ Humans will take continuous damage instead.
 /datum/component/healing_reduction/process(delta_time)
 	if(!parent)
 		qdel(src)
+		return
+
 	healing_reduction = max(healing_reduction - healing_reduction_dissipation * delta_time, 0)
 
+	if(healing_reduction <= 0)
+		qdel(src)
+		return
+
 	if(ishuman(parent)) //deals brute to humans
 		var/mob/living/carbon/human/H = parent
 		H.apply_damage(healing_reduction_dissipation * delta_time, BRUTE)
```

Comment:
```suggestion
		var/mob/living/carbon/human/human_parent = parent
		human_parent.apply_damage(healing_reduction_dissipation * delta_time, BRUTE)
```

Reply:
Since we're already here, should bring this code in lines w/ the code style guidelines: https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#use-descriptive-and-obvious-names


`A` below could also be atom_parent; but its outside of the context for me to make it as a suggestion.

Reply:
can I make comment like this on PRs of others? if so how?

Reply:
See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review